### PR TITLE
Fix commitstatus summary

### PR DESCRIPTION
**Pull request by lunny**
The target_url is necessary for the UI, but missed in commit_status_summary table. This PR fix it.

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

**Comment by wxiaoguang** [comment]
Unrelated changes from ` Fix rename branch 500 when the target branch is deleted but exist in database  `

File: models/migrations/migrations.go [hunk 1]
```diff
@@ -580,6 +580,8 @@ var migrations = []Migration{
 	NewMigration("Add unique index for project issue table", v1_23.AddUniqueIndexForProjectIssue),
 	// v295 -> v296
 	NewMigration("Add commit status summary table", v1_23.AddCommitStatusSummary),
+	// v296 -> v297
+	NewMigration("Add missed field of commit status summary table", v1_23.AddCommitStatusSummary2),
```
Comment (silverwind) [inline models/migrations/migrations.go]:
```suggestion
	NewMigration("Add missing field of commit status summary table", v1_23.AddCommitStatusSummary2),
```

Better english. Also see [here](https://www.bbc.co.uk/worldservice/learningenglish/grammar/learnit/learnitv188.shtml).

Comment (silverwind) [reply models/migrations/migrations.go]: Not resolved.

File: models/migrations/v1_23/v296.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import "xorm.io/xorm"
+
+func AddCommitStatusSummary2(x *xorm.Engine) error {
+	type CommitStatusSummary struct {
+		ID        int64  `xorm:"pk autoincr"`
+		TargetURL string `xorm:"TEXT"`
+	}
+	// there is no migrations because if there is no data on this table, it will fall back to get data
+	// from commit status
+	return x.Sync2(new(CommitStatusSummary))
```
Comment (delvh) [inline models/migrations/v1_23/v296.go]: I thought `Sync2` is deprecated?
```suggestion
	return x.Sync(new(CommitStatusSummary))
```

**Review by delvh — APPROVED** [review]

**Pull request merged**